packages/admin-ui-extensions/src/components/Form/examples/Form.example.ts

```diff
@@ -0,0 +1,17 @@
+import {extend, ExtensionPoint, TextField, Form} from '@shopify/admin-ui-extensions';
+
+extend('Playground', (root) => {
+  const form = root.createComponent(Form, {
+    onSubmit: () => console.log('submitted!'),
+  }, [
+    root.createComponent(TextField, {
```

Comment:
we need to assign this component to a variable and append it to the form component, right?

Reply:
oh! gotcha, I didn't realize remote-ui could do that. I still think it would be good to assign the component to a variable for readability, but that's just nitpicking now